### ISSUE TITLE
Ubuntu2204 - Update default apt mirrors

### ISSUE THREAD

**Issue by maroskukan** [maroskukan/packer-cookbook#6]
Investigate how apt mirrors are set during the installation and possible change them to global instead of local or regional ones.
https://askubuntu.com/questions/1326043/how-to-add-apt-repository-with-ubuntu-autoinstall

**Comment by maroskukan** [comment]
The following settings in user-data could affect the mirror selection
```yml
  apt:
      geoip: false
      primary:
          - arches: [default]
            uri: https://mirrors.edge.kernel.org/ubuntu/
      security:
          - arches: [default]
            uri: https://mirrors.edge.kernel.org/ubuntu/
```

**Comment by maroskukan** [comment]
Implemented in #11
